server/data/statementsClient.ts

```diff
@@ -9,7 +9,7 @@ import type {
   StatementUpdate,
 } from './statementsClientTypes'
 import { Status } from '../services/statementServiceTypes'
-import type { DraftInvolvedStaff } from '../services/report/draftReportService'
+import type { DraftInvolvedStaff } from '../services/report/draftInvolvedStaffService'
```

Comment:
data layer relying on service layer - need to move this stuff down or create a `model`